Xwt/Xwt.Accessibility/Accessible.cs

```diff
@@ -120,6 +120,12 @@ internal Accessible (object nativeParent)
 			}
 		}
 
+		public Widget LabeledBy {
```

Comment:
This naming is a little bit confusing, I think something like `LabelWidget` would be better. @iainx afair there was something similar in Cocoa, do you remember how it works?

Reply:
Also why doesn't it have a getter?

Reply:
There are two properties in Cocoa

`AccessibilityTitleUIElement` which links the control element to its title control
`AccessibilityServesAsTitleForUIElements` which links the title control to an array of control elements

So `LabeledBy` would be the equivalent of `AccessibilityTitleUIElement`

Reply:
Cool, I'll change the name and add the Cocoa side.

No getter as there is no way to map the Xwt.Widget back in a practical way from the underlying implementation. I can however store the value in `Accessible` directly if that's useful?

Reply:
@garuma yeah, but in this case there is no way from WPF to Xwt (or do some WPF widgets have a predefined `LabeldBy` UIElement?), so to be aligned with the rest of the API there should be a getter and we could store a reference in `Accessible`. This would make it possible to check whether there is one already assigned and which one.